src/components/layout/authLayout/styles.js

```diff
@@ -0,0 +1,63 @@
+import styled from 'styled-components'
+
+export const Layout = styled.div`
+  width: 100%;
+  min-height: 100vh;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  background-color: ${({ theme }) => theme.COLORS.GRAY};
+  color: ${({ theme }) => theme.COLORS.WHITE};
+
+  div {
+    display: flex;
+    flex-direction: column;
+    gap: 5rem;
+
+    margin: 7rem 2rem;
+    animation: fromLeftToRight 0.8s;
+  }
+
+  hr {
+    background-color: #3f445a;
+    border: none;
+    width: 100%;
+    height: 0.1rem;
+  }
+
+  @keyframes fromLeftToRight {
```

Comment:
Analisar se essa animação vai se repetir em alguma outra parte do sistema, se sim, é bom fazer um arquivo `animations.js` no styles globais e importar elas quando for precisar.

Exemplo:

- arquivo `animations.js`: https://github.dev/Elias-Neto/food-explorer-frontend
- importando uma animação (linhas 2 e 14): https://github.dev/Elias-Neto/food-explorer-frontend

Reply:
A princípio acredito que não se repita